ios/FluentUI/Vnext/Notification/MSFNotificationView.swift

```diff
@@ -15,7 +15,7 @@ import UIKit
     ///   - message: The primary text to display in the Notification.
     @objc public init(style: MSFNotificationStyle,
                       message: String) {
-        notification = NotificationViewSwiftUI(style: style, message: message)
+        notification = NotificationViewSwiftUI(style: style, shouldPresentItself: false, message: message)
```

Comment:
Agreed! I think just "Notification" would be good. This change replaces the UIKit implementation (currently named NotificationView), so can we delete that implementation in this CR to avoid confusion? Or do we need to keep it just in case?